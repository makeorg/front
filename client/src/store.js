/* @flow */

import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import Cookies from 'universal-cookie';
import { SESSION_ID_COOKIE_KEY } from 'Src/constants/config';
import { uuid } from 'Src/helpers/uuid';
import ApiService from 'Src/api/ApiService';
import DateHelper from 'Src/helpers/date';
import rootReducer from './reducers';

export default function configureStore(initialState: Object = {}) {
  const composeEnhancers = (typeof window !== 'undefined' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose;

  const cookies = new Cookies();
  let sessionId = cookies.get(SESSION_ID_COOKIE_KEY);

  if (!sessionId) {
    sessionId = uuid();
    cookies.set(SESSION_ID_COOKIE_KEY, sessionId);
  }

  ApiService.sessionId = sessionId;

  if (initialState.sequence && initialState.sequence.question) {
    ApiService.questionId = initialState.sequence.question.questionId;
    ApiService.operationId = initialState.sequence.question.operationId;
  }
  ApiService.source = initialState.appConfig.source;
  ApiService.country = initialState.appConfig.country;
  ApiService.language = initialState.appConfig.language;
  DateHelper.language = initialState.appConfig.language;

  return createStore(
    rootReducer,
    initialState,
    composeEnhancers(applyMiddleware(thunk))
  );
}
