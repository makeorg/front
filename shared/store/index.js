import { UserApiService } from 'Shared/api/UserApiService';

/* @flow */

import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import { rootReducer } from './reducers';

export function configureStore(initialState: Object = {}) {
  const composeEnhancers =
    (typeof window !== 'undefined' &&
      window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) ||
    compose;

  return createStore(
    rootReducer,
    initialState,
    composeEnhancers(applyMiddleware(thunk))
  );
}

export const authenticationState = async () => {
  let authentificationState;
  try {
    const user = await UserApiService.me();
    authentificationState = {
      isLoggedIn: !!user,
      user,
    };
  } catch (error) {
    authentificationState = {
      isLoggedIn: false,
      user: undefined,
    };
  }

  return authentificationState;
};
