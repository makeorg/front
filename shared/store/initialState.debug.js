export const initialStateDebug = {
  appConfig: {
    source: 'core',
    language: 'fr',
    country: 'FR',
    translations: require('../../server/staticData/i18n/fr-FR.json'),
  },
  user: {
    authentification: {
      errors: [],
      isLoggedIn: false,
      user: undefined,
    },
  },
  proposal: {
    hasProposed: false,
  },
  sequence: {
    /* 
    questionSlug: 'mieuxmanger',
    */
    votedProposalIds: [],
  },
  currentQuestion: undefined,
  questions: {
    /* '8358bb5d-493c-4ae8-88be-3de613e2e527': {
      questionId: '8358bb5d-493c-4ae8-88be-3de613e2e527',
      operationId: '65dfe694-2ea9-486b-8bc5-3107316fa6ff'
    } */
  },
};
