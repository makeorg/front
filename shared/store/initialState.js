// @flow
import { type StateRoot } from './types';

export const initialState: StateRoot = {
  appConfig: {
    source: '',
    language: '',
    country: '',
    translations: {},
    queryParams: {},
  },
  proposal: {
    hasProposed: false,
    popularProposals: [],
    error: undefined,
    data: undefined,
  },
  sequence: {
    isSequenceCollapsed: false,
    firstProposal: undefined,
    questionSlug: undefined,
    currentIndex: 0,
    votedProposalIds: {},
    proposals: [],
  },
  questions: {},
  currentQuestion: '',
  notification: {
    contentType: undefined,
  },
  user: {
    authentication: {
      errors: [],
      isLoggedIn: false,
      user: undefined,
    },
    passwordRecovery: {
      newPassword: undefined,
      resetToken: undefined,
      userId: undefined,
      errorMessage: undefined,
      error: false,
      updated: false,
    },
  },
  modal: {
    isOpen: false,
    contentType: '',
    showExpirationSession: false,
  },
  partners: {},
};

export const createInitialState = (): StateRoot =>
  JSON.parse(JSON.stringify(initialState));
