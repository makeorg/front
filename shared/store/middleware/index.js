// @flow

import thunk from 'redux-thunk';
import { requestContext } from './requestContext';
import { question } from './question';
import { countryLanguage } from './countryLanguage';

export const middlewares = [thunk, requestContext, question, countryLanguage];
