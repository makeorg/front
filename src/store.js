/* @flow */

import { createStore, applyMiddleware, compose } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from './reducers';
import ApiService from './api/ApiService';
import DateHelper from './helpers/date';

export default function configureStore(initialState: Object = {}) {
  const composeEnhancers = (typeof window !== 'undefined' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__) || compose;

  ApiService.operationId = initialState.appConfig.operationId;
  ApiService.source = initialState.appConfig.source;
  ApiService.country = initialState.appConfig.country;
  ApiService.language = initialState.appConfig.language;
  DateHelper.language = initialState.appConfig.language;

  return createStore(
    rootReducer,
    initialState,
    composeEnhancers(applyMiddleware(thunk))
  );
}
