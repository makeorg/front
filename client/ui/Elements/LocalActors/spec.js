import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';

import { configureStore } from 'Shared/store';
import { LocalActors } from './index';

describe('LocalActors', () => {
  it('snapshot by default ', () => {
    const { container } = render(
      <Provider store={configureStore()}>
        <LocalActors
          questionId="66a9230b-08cb-4f37-8ed8-aa95a8eac19a"
          slug="environnement"
        />
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });

  it('snapshot with data', () => {
    const { container } = render(
      <Provider
        store={configureStore({
          appConfig: { country: 'FR', language: 'fr' },
          partners: {
            isLoading: false,
            environnement: {
              actors: {
                total: 2,
                results: [
                  {
                    organisationId: '12b522207-ffa5-471b-a13d-d836793694ac',
                    organisationName: 'toto orga',
                    slug: 'toto-orga',
                    avatarUrl: 'string',
                    description: 'string',
                    publicProfile: true,
                    proposalsCount: 0,
                    votesCount: 1,
                    language: 'fr',
                    country: 'FR',
                  },
                  {
                    organisationId: '26eaf9e87-8d45-4c9d-8dde-29671ce607ff',
                    organisationName: 'CCAH',
                    slug: 'ccah',
                    avatarUrl: null,
                    description: null,
                    publicProfile: true,
                    proposalsCount: null,
                    votesCount: null,
                    language: 'fr',
                    country: 'FR',
                  },
                  {
                    organisationId: '32b522207-ffa5-471b-a13d-d836793694ac',
                    organisationName: 'toto orga',
                    slug: 'toto-orga',
                    avatarUrl: 'string',
                    description: 'string',
                    publicProfile: true,
                    proposalsCount: 0,
                    votesCount: 1,
                    language: 'fr',
                    country: 'FR',
                  },
                  {
                    organisationId: '46eaf9e87-8d45-4c9d-8dde-29671ce607ff',
                    organisationName: 'CCAH',
                    slug: 'ccah',
                    avatarUrl: null,
                    description: null,
                    publicProfile: true,
                    proposalsCount: null,
                    votesCount: null,
                    language: 'fr',
                    country: 'FR',
                  },
                  {
                    organisationId: '52b522207-ffa5-471b-a13d-d836793694ac',
                    organisationName: 'toto orga',
                    slug: 'toto-orga',
                    avatarUrl: 'string',
                    description: 'string',
                    publicProfile: true,
                    proposalsCount: 0,
                    votesCount: 1,
                    language: 'fr',
                    country: 'FR',
                  },
                ],
              },
            },
          },
        })}
      >
        <BrowserRouter>
          <LocalActors
            questionId="66a9230b-08cb-4f37-8ed8-aa95a8eac19a"
            slug="environnement"
          />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
