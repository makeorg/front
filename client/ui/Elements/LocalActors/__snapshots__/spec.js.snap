// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocalActors snapshot by default  1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  height: 100%;
}

.c2 {
  display: block;
  position: relative;
  width: 4rem;
  height: 4rem;
  overflow: hidden;
}

.c3 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
}

.c4 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
  -webkit-animation-delay: -0.45s;
  animation-delay: -0.45s;
}

.c5 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
  -webkit-animation-delay: -0.3s;
  animation-delay: -0.3s;
}

.c6 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
  -webkit-animation-delay: -0.15s;
  animation-delay: -0.15s;
}

.c0 {
  padding: 0;
  list-style: none;
}

.c7 {
  font-size: 12px;
  line-height: 18px;
  color: rgb(74,74,74);
  margin-top: 20px;
}

@media (min-width:360px) {
  .c7 {
    font-size: 14px;
    line-height: 21px;
  }
}

@media (min-width:969px) {
  .c7 {
    font-size: 16px;
    line-height: 22px;
  }
}

<div>
  <ul
    class="c0"
  >
    <aside
      aria-label="common.loading"
      class="f9ghfa-0 f9ghfa-2 f9ghfa-6 f9ghfa-7 c1"
    >
      <div
        class="c2"
      >
        <div
          aria-hidden="true"
          class="c3"
        />
        <div
          aria-hidden="true"
          class="c4"
        />
        <div
          aria-hidden="true"
          class="c5"
        />
        <div
          aria-hidden="true"
          class="c6"
        />
      </div>
    </aside>
    <p
      class="sc-1nc0118-0 c7"
    >
      consultation.local_actors.participate
    </p>
  </ul>
</div>
`;

exports[`LocalActors snapshot with data 1`] = `
.c3 {
  margin-right: 10px;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.c3 svg,
.c3 img {
  border-radius: 50%;
  overflow: hidden;
  border: 1px solid rgba(0,0,0,0.5);
  background-color: rgb(255,255,255);
  min-width: 30px;
}

.c4 {
  color: rgb(222,26,66);
  font-size: 12px;
}

.c4:hover,
.c4:focus {
  color: rgb(222,26,66);
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  width: 100%;
  height: 100%;
}

.c6 {
  display: block;
  position: relative;
  width: 4rem;
  height: 4rem;
  overflow: hidden;
}

.c7 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
}

.c8 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
  -webkit-animation-delay: -0.45s;
  animation-delay: -0.45s;
}

.c9 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
  -webkit-animation-delay: -0.3s;
  animation-delay: -0.3s;
}

.c10 {
  box-sizing: border-box;
  display: block;
  position: absolute;
  width: 3.188rem;
  height: 3.188rem;
  margin: 0.375rem;
  border: 0.375rem solid rgb(178,178,178);
  border-radius: 50%;
  -webkit-animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  animation: zdZhV 1.2s cubic-bezier(0.5,0,0.5,1) infinite;
  border-color: rgb(178,178,178) transparent transparent transparent;
  -webkit-animation-delay: -0.15s;
  animation-delay: -0.15s;
}

.c0 {
  padding: 0;
  list-style: none;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: rgb(83,83,83);
  font-size: 12px;
  padding: 20px 0;
  border-bottom: 1px solid rgb(229,229,229);
}

.c1:nth-child(1) {
  padding-top: 4px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-right: 5px;
}

.c11 {
  font-size: 12px;
  line-height: 18px;
  color: rgb(74,74,74);
  margin-top: 20px;
}

@media (min-width:360px) {
  .c4 {
    font-size: 14px;
  }
}

@media (min-width:360px) {
  .c4 {
    font-size: 16px;
  }
}

@media (min-width:360px) {
  .c11 {
    font-size: 14px;
    line-height: 21px;
  }
}

@media (min-width:969px) {
  .c11 {
    font-size: 16px;
    line-height: 22px;
  }
}

<div>
  <ul
    class="c0"
  >
    <li
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
        >
          <img
            alt="toto orga"
            height="30"
            src="string"
            width="30"
          />
        </span>
      </div>
      <div>
        <div>
          <a
            class="c4"
            href="/FR-fr/profile/toto orga"
          >
            toto orga
          </a>
          <svg
            height="1em"
            style="font-size: 14px; fill: rgb(74, 144, 226); margin-left: 5px;"
            viewBox="0 0 438.533 438.533"
            width="1em"
          >
            <path
              d="M409.133 109.203c-19.608-33.592-46.205-60.189-79.798-79.796C295.736 9.801 259.058 0 219.273 0c-39.781 0-76.47 9.801-110.063 29.407-33.595 19.604-60.192 46.201-79.8 79.796C9.801 142.8 0 179.489 0 219.267c0 39.78 9.804 76.463 29.407 110.062 19.607 33.592 46.204 60.189 79.799 79.798 33.597 19.605 70.283 29.407 110.063 29.407s76.47-9.802 110.065-29.407c33.593-19.602 60.189-46.206 79.795-79.798 19.603-33.596 29.403-70.284 29.403-110.062.001-39.782-9.8-76.472-29.399-110.064zm-47.688 76.66L206.42 340.889c-3.617 3.62-7.992 5.428-13.134 5.428-4.948 0-9.229-1.808-12.847-5.428L77.083 237.539c-3.422-3.429-5.137-7.703-5.137-12.847 0-5.328 1.709-9.709 5.137-13.136l25.981-25.693c3.621-3.616 7.898-5.424 12.85-5.424s9.235 1.809 12.85 5.424l64.525 64.523 116.485-116.199c3.617-3.617 7.898-5.426 12.847-5.426 4.945 0 9.233 1.809 12.847 5.426l25.981 25.697c3.432 3.424 5.14 7.801 5.14 13.129 0 5.141-1.708 9.424-5.144 12.85z"
            />
          </svg>
        </div>
        <div>
          consultation.local_actors.proposals
           . 
          consultation.local_actors.votes
        </div>
      </div>
    </li>
    <li
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
        >
          <svg
            aria-hidden="true"
            height="30"
            viewBox="0 0 30 30"
            width="30"
          >
            <defs>
              <circle
                cx="15"
                cy="15"
                r="15"
              />
              <path
                d="M17 15.24c0 .757-.228 1.406-.684 1.947-.455.542-1.004.813-1.646.813H5.33c-.642 0-1.19-.27-1.646-.813C3.228 16.646 3 15.998 3 15.24c0-.59.031-1.148.093-1.672a8.612 8.612 0 0 1 .345-1.584c.167-.53.38-.986.64-1.364a3.005 3.005 0 0 1 1.027-.927c.427-.24.917-.36 1.472-.36.955.89 2.096 1.334 3.423 1.334s2.468-.445 3.423-1.334c.555 0 1.045.12 1.472.36.426.24.769.548 1.028.927.259.378.472.833.64 1.364.167.532.282 1.06.344 1.584S17 14.649 17 15.24zM12.964 3.177C13.788 3.955 14.2 4.896 14.2 6c0 1.104-.41 2.047-1.23 2.828C12.15 9.61 11.16 10 10 10c-1.16 0-2.15-.39-2.97-1.172C6.21 8.047 5.8 7.104 5.8 6c0-1.104.41-2.047 1.23-2.828C7.85 2.39 8.84 2 10 2c1.16 0 2.147.392 2.964 1.177z"
              />
            </defs>
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <mask
                fill="#fff"
              >
                <circle
                  cx="15"
                  cy="15"
                  r="15"
                />
              </mask>
              <circle
                cx="15"
                cy="15"
                fill="#FFF"
                r="15"
              />
              <circle
                cx="15"
                cy="15"
                fill="#000"
                fill-opacity="0.1"
                r="14.5"
                stroke="#000"
                stroke-linejoin="square"
                stroke-opacity="0.2"
              />
              <g>
                <g
                  transform="translate(5 5)"
                >
                  <mask
                    fill="#fff"
                  >
                    <path
                      d="M17 15.24c0 .757-.228 1.406-.684 1.947-.455.542-1.004.813-1.646.813H5.33c-.642 0-1.19-.27-1.646-.813C3.228 16.646 3 15.998 3 15.24c0-.59.031-1.148.093-1.672a8.612 8.612 0 0 1 .345-1.584c.167-.53.38-.986.64-1.364a3.005 3.005 0 0 1 1.027-.927c.427-.24.917-.36 1.472-.36.955.89 2.096 1.334 3.423 1.334s2.468-.445 3.423-1.334c.555 0 1.045.12 1.472.36.426.24.769.548 1.028.927.259.378.472.833.64 1.364.167.532.282 1.06.344 1.584S17 14.649 17 15.24zM12.964 3.177C13.788 3.955 14.2 4.896 14.2 6c0 1.104-.41 2.047-1.23 2.828C12.15 9.61 11.16 10 10 10c-1.16 0-2.15-.39-2.97-1.172C6.21 8.047 5.8 7.104 5.8 6c0-1.104.41-2.047 1.23-2.828C7.85 2.39 8.84 2 10 2c1.16 0 2.147.392 2.964 1.177z"
                    />
                  </mask>
                  <path
                    d="M17 15.24c0 .757-.228 1.406-.684 1.947-.455.542-1.004.813-1.646.813H5.33c-.642 0-1.19-.27-1.646-.813C3.228 16.646 3 15.998 3 15.24c0-.59.031-1.148.093-1.672a8.612 8.612 0 0 1 .345-1.584c.167-.53.38-.986.64-1.364a3.005 3.005 0 0 1 1.027-.927c.427-.24.917-.36 1.472-.36.955.89 2.096 1.334 3.423 1.334s2.468-.445 3.423-1.334c.555 0 1.045.12 1.472.36.426.24.769.548 1.028.927.259.378.472.833.64 1.364.167.532.282 1.06.344 1.584S17 14.649 17 15.24zM12.964 3.177C13.788 3.955 14.2 4.896 14.2 6c0 1.104-.41 2.047-1.23 2.828C12.15 9.61 11.16 10 10 10c-1.16 0-2.15-.39-2.97-1.172C6.21 8.047 5.8 7.104 5.8 6c0-1.104.41-2.047 1.23-2.828C7.85 2.39 8.84 2 10 2c1.16 0 2.147.392 2.964 1.177z"
                    fill="#000"
                    fill-opacity="0.3"
                    fill-rule="nonzero"
                  />
                  <mask
                    fill="#fff"
                  >
                    <path
                      d="M0 0h20v20H0z"
                      fill="#6A6A6A"
                    />
                  </mask>
                </g>
              </g>
            </g>
          </svg>
        </span>
      </div>
      <div>
        <div>
          <a
            class="c4"
            href="/FR-fr/profile/CCAH"
          >
            CCAH
          </a>
          <svg
            height="1em"
            style="font-size: 14px; fill: rgb(74, 144, 226); margin-left: 5px;"
            viewBox="0 0 438.533 438.533"
            width="1em"
          >
            <path
              d="M409.133 109.203c-19.608-33.592-46.205-60.189-79.798-79.796C295.736 9.801 259.058 0 219.273 0c-39.781 0-76.47 9.801-110.063 29.407-33.595 19.604-60.192 46.201-79.8 79.796C9.801 142.8 0 179.489 0 219.267c0 39.78 9.804 76.463 29.407 110.062 19.607 33.592 46.204 60.189 79.799 79.798 33.597 19.605 70.283 29.407 110.063 29.407s76.47-9.802 110.065-29.407c33.593-19.602 60.189-46.206 79.795-79.798 19.603-33.596 29.403-70.284 29.403-110.062.001-39.782-9.8-76.472-29.399-110.064zm-47.688 76.66L206.42 340.889c-3.617 3.62-7.992 5.428-13.134 5.428-4.948 0-9.229-1.808-12.847-5.428L77.083 237.539c-3.422-3.429-5.137-7.703-5.137-12.847 0-5.328 1.709-9.709 5.137-13.136l25.981-25.693c3.621-3.616 7.898-5.424 12.85-5.424s9.235 1.809 12.85 5.424l64.525 64.523 116.485-116.199c3.617-3.617 7.898-5.426 12.847-5.426 4.945 0 9.233 1.809 12.847 5.426l25.981 25.697c3.432 3.424 5.14 7.801 5.14 13.129 0 5.141-1.708 9.424-5.144 12.85z"
            />
          </svg>
        </div>
        <div>
          consultation.local_actors.proposals
           . 
          consultation.local_actors.votes
        </div>
      </div>
    </li>
    <li
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
        >
          <img
            alt="toto orga"
            height="30"
            src="string"
            width="30"
          />
        </span>
      </div>
      <div>
        <div>
          <a
            class="c4"
            href="/FR-fr/profile/toto orga"
          >
            toto orga
          </a>
          <svg
            height="1em"
            style="font-size: 14px; fill: rgb(74, 144, 226); margin-left: 5px;"
            viewBox="0 0 438.533 438.533"
            width="1em"
          >
            <path
              d="M409.133 109.203c-19.608-33.592-46.205-60.189-79.798-79.796C295.736 9.801 259.058 0 219.273 0c-39.781 0-76.47 9.801-110.063 29.407-33.595 19.604-60.192 46.201-79.8 79.796C9.801 142.8 0 179.489 0 219.267c0 39.78 9.804 76.463 29.407 110.062 19.607 33.592 46.204 60.189 79.799 79.798 33.597 19.605 70.283 29.407 110.063 29.407s76.47-9.802 110.065-29.407c33.593-19.602 60.189-46.206 79.795-79.798 19.603-33.596 29.403-70.284 29.403-110.062.001-39.782-9.8-76.472-29.399-110.064zm-47.688 76.66L206.42 340.889c-3.617 3.62-7.992 5.428-13.134 5.428-4.948 0-9.229-1.808-12.847-5.428L77.083 237.539c-3.422-3.429-5.137-7.703-5.137-12.847 0-5.328 1.709-9.709 5.137-13.136l25.981-25.693c3.621-3.616 7.898-5.424 12.85-5.424s9.235 1.809 12.85 5.424l64.525 64.523 116.485-116.199c3.617-3.617 7.898-5.426 12.847-5.426 4.945 0 9.233 1.809 12.847 5.426l25.981 25.697c3.432 3.424 5.14 7.801 5.14 13.129 0 5.141-1.708 9.424-5.144 12.85z"
            />
          </svg>
        </div>
        <div>
          consultation.local_actors.proposals
           . 
          consultation.local_actors.votes
        </div>
      </div>
    </li>
    <li
      class="c1"
    >
      <div
        class="c2"
      >
        <span
          class="c3"
        >
          <svg
            aria-hidden="true"
            height="30"
            viewBox="0 0 30 30"
            width="30"
          >
            <defs>
              <circle
                cx="15"
                cy="15"
                r="15"
              />
              <path
                d="M17 15.24c0 .757-.228 1.406-.684 1.947-.455.542-1.004.813-1.646.813H5.33c-.642 0-1.19-.27-1.646-.813C3.228 16.646 3 15.998 3 15.24c0-.59.031-1.148.093-1.672a8.612 8.612 0 0 1 .345-1.584c.167-.53.38-.986.64-1.364a3.005 3.005 0 0 1 1.027-.927c.427-.24.917-.36 1.472-.36.955.89 2.096 1.334 3.423 1.334s2.468-.445 3.423-1.334c.555 0 1.045.12 1.472.36.426.24.769.548 1.028.927.259.378.472.833.64 1.364.167.532.282 1.06.344 1.584S17 14.649 17 15.24zM12.964 3.177C13.788 3.955 14.2 4.896 14.2 6c0 1.104-.41 2.047-1.23 2.828C12.15 9.61 11.16 10 10 10c-1.16 0-2.15-.39-2.97-1.172C6.21 8.047 5.8 7.104 5.8 6c0-1.104.41-2.047 1.23-2.828C7.85 2.39 8.84 2 10 2c1.16 0 2.147.392 2.964 1.177z"
              />
            </defs>
            <g
              fill="none"
              fill-rule="evenodd"
            >
              <mask
                fill="#fff"
              >
                <circle
                  cx="15"
                  cy="15"
                  r="15"
                />
              </mask>
              <circle
                cx="15"
                cy="15"
                fill="#FFF"
                r="15"
              />
              <circle
                cx="15"
                cy="15"
                fill="#000"
                fill-opacity="0.1"
                r="14.5"
                stroke="#000"
                stroke-linejoin="square"
                stroke-opacity="0.2"
              />
              <g>
                <g
                  transform="translate(5 5)"
                >
                  <mask
                    fill="#fff"
                  >
                    <path
                      d="M17 15.24c0 .757-.228 1.406-.684 1.947-.455.542-1.004.813-1.646.813H5.33c-.642 0-1.19-.27-1.646-.813C3.228 16.646 3 15.998 3 15.24c0-.59.031-1.148.093-1.672a8.612 8.612 0 0 1 .345-1.584c.167-.53.38-.986.64-1.364a3.005 3.005 0 0 1 1.027-.927c.427-.24.917-.36 1.472-.36.955.89 2.096 1.334 3.423 1.334s2.468-.445 3.423-1.334c.555 0 1.045.12 1.472.36.426.24.769.548 1.028.927.259.378.472.833.64 1.364.167.532.282 1.06.344 1.584S17 14.649 17 15.24zM12.964 3.177C13.788 3.955 14.2 4.896 14.2 6c0 1.104-.41 2.047-1.23 2.828C12.15 9.61 11.16 10 10 10c-1.16 0-2.15-.39-2.97-1.172C6.21 8.047 5.8 7.104 5.8 6c0-1.104.41-2.047 1.23-2.828C7.85 2.39 8.84 2 10 2c1.16 0 2.147.392 2.964 1.177z"
                    />
                  </mask>
                  <path
                    d="M17 15.24c0 .757-.228 1.406-.684 1.947-.455.542-1.004.813-1.646.813H5.33c-.642 0-1.19-.27-1.646-.813C3.228 16.646 3 15.998 3 15.24c0-.59.031-1.148.093-1.672a8.612 8.612 0 0 1 .345-1.584c.167-.53.38-.986.64-1.364a3.005 3.005 0 0 1 1.027-.927c.427-.24.917-.36 1.472-.36.955.89 2.096 1.334 3.423 1.334s2.468-.445 3.423-1.334c.555 0 1.045.12 1.472.36.426.24.769.548 1.028.927.259.378.472.833.64 1.364.167.532.282 1.06.344 1.584S17 14.649 17 15.24zM12.964 3.177C13.788 3.955 14.2 4.896 14.2 6c0 1.104-.41 2.047-1.23 2.828C12.15 9.61 11.16 10 10 10c-1.16 0-2.15-.39-2.97-1.172C6.21 8.047 5.8 7.104 5.8 6c0-1.104.41-2.047 1.23-2.828C7.85 2.39 8.84 2 10 2c1.16 0 2.147.392 2.964 1.177z"
                    fill="#000"
                    fill-opacity="0.3"
                    fill-rule="nonzero"
                  />
                  <mask
                    fill="#fff"
                  >
                    <path
                      d="M0 0h20v20H0z"
                      fill="#6A6A6A"
                    />
                  </mask>
                </g>
              </g>
            </g>
          </svg>
        </span>
      </div>
      <div>
        <div>
          <a
            class="c4"
            href="/FR-fr/profile/CCAH"
          >
            CCAH
          </a>
          <svg
            height="1em"
            style="font-size: 14px; fill: rgb(74, 144, 226); margin-left: 5px;"
            viewBox="0 0 438.533 438.533"
            width="1em"
          >
            <path
              d="M409.133 109.203c-19.608-33.592-46.205-60.189-79.798-79.796C295.736 9.801 259.058 0 219.273 0c-39.781 0-76.47 9.801-110.063 29.407-33.595 19.604-60.192 46.201-79.8 79.796C9.801 142.8 0 179.489 0 219.267c0 39.78 9.804 76.463 29.407 110.062 19.607 33.592 46.204 60.189 79.799 79.798 33.597 19.605 70.283 29.407 110.063 29.407s76.47-9.802 110.065-29.407c33.593-19.602 60.189-46.206 79.795-79.798 19.603-33.596 29.403-70.284 29.403-110.062.001-39.782-9.8-76.472-29.399-110.064zm-47.688 76.66L206.42 340.889c-3.617 3.62-7.992 5.428-13.134 5.428-4.948 0-9.229-1.808-12.847-5.428L77.083 237.539c-3.422-3.429-5.137-7.703-5.137-12.847 0-5.328 1.709-9.709 5.137-13.136l25.981-25.693c3.621-3.616 7.898-5.424 12.85-5.424s9.235 1.809 12.85 5.424l64.525 64.523 116.485-116.199c3.617-3.617 7.898-5.426 12.847-5.426 4.945 0 9.233 1.809 12.847 5.426l25.981 25.697c3.432 3.424 5.14 7.801 5.14 13.129 0 5.141-1.708 9.424-5.144 12.85z"
            />
          </svg>
        </div>
        <div>
          consultation.local_actors.proposals
           . 
          consultation.local_actors.votes
        </div>
      </div>
    </li>
    <aside
      aria-label="common.loading"
      class="f9ghfa-0 f9ghfa-2 f9ghfa-6 f9ghfa-7 c5"
    >
      <div
        class="c6"
      >
        <div
          aria-hidden="true"
          class="c7"
        />
        <div
          aria-hidden="true"
          class="c8"
        />
        <div
          aria-hidden="true"
          class="c9"
        />
        <div
          aria-hidden="true"
          class="c10"
        />
      </div>
    </aside>
    <p
      class="sc-1nc0118-0 c11"
    >
      consultation.local_actors.participate
    </p>
  </ul>
</div>
`;
